tensorflow/contrib/lite/tools/benchmark/README.md

```diff
@@ -133,7 +133,7 @@ For example, to compile with profiling support on Android, add this flag to the
 
 ```
 bazel build -c opt \
-  --config=android_arm \
+  --config=android_arm64 \
```

Comment:
Instead of adding it here, can you add a line that use android_arm64 is your Phone is 64 bit. E.g. Pixel, Pixel2 and many of the new Android phones.